src/coreclr/System.Private.CoreLib/src/System/Reflection/Emit/RuntimeLocalBuilder.cs

```diff
@@ -24,21 +24,11 @@ internal LocalBuilder(int localIndex, Type localType, MethodInfo methodBuilder,
         }
         #endregion
 
-        #region Internal Members
-        internal int GetLocalIndex()
-        {
-            return m_localIndex;
-        }
-        internal MethodInfo GetMethodBuilder()
-        {
-            return m_methodBuilder;
-        }
-        #endregion
-
         #region LocalVariableInfo Override
         public override bool IsPinned => m_isPinned;
         public override Type LocalType => m_localType;
         public override int LocalIndex => m_localIndex;
+        public override MethodInfo Method => m_methodBuilder;
```

Comment:
Unsure of the policy for `region` tags, but this likely shouldn't be in the `LocalVariableInfo` section. Should we create one for `LocalBuilder` or remove them entirely?